openmc/surface.py

```diff
@@ -175,6 +174,14 @@ def clone(self, memo=None):
 
         return memo[self]
 
+    @abstractmethod
+    def evaluate(self, point):
```

Comment:
No new functionality here, you just realized this wasnt abstract, right?

Reply:
Correct!